src/EditorFeatures/CSharpTest/RemoveUnnecessarySuppression/RemoveUnnecessarySuppressionTests.cs

```diff
@@ -0,0 +1,207 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Can all the analyzer/fixer/tests files added here be moved to shared projects under `src\Analyzers`?

Reply:
done!

Reply:
I see this resolved but the files are still in `src\Features` and `src\EditorFeatures`, not under `src\Analyzers`. Do you have another commit to follow?